circle.yml

```diff
@@ -0,0 +1,12 @@
+dependencies:
+  override:
+    - printf '%s\n' color=false progress=false >~/.npmrc
+    - make setup
+
+machine:
+  node:
+    version: 4
```

Comment:
Excellent point! I had not considered that. I'll install Node 6 on the server. :)